man/man8/zinject.8

```diff
@@ -112,6 +112,9 @@ Specify
 .BR "dtl" " for an ECHILD error,"
 .BR "io" " for an EIO error where reopening the device will succeed, or"
 .BR "nxio" " for an ENXIO error where reopening the device will fail."
+For EIO and ENXIO, the "failed" reads or writes still occur.  The probe simply
+sets the value of zio->io_error so the consumer of the zio believes the read or
```

Comment:
Let's rephrase this in a way which doesn't reference internal data structures or their members.  (in this case zio->io_error).

Reply:
Done.